src/fits.cc

```diff
@@ -720,12 +720,11 @@ void writeKeyFromProperty(
         }
     } else {
         // FIXME: inherited this error handling from fitsIo.cc; need a better option.
-        pex::logging::Log::getDefaultLog().log(
-            pex::logging::Log::WARN,
+        LOGLS_WARN("afw.writeKeyFromProperty",
             makeErrorMessage(
                 fits.fptr, fits.status,
-                boost::format("In %s, unknown type '%s' for key '%s'.")
-                % BOOST_CURRENT_FUNCTION % valueType.name() % key
+                boost::format("Unknown type '%s' for key '%s'.")
+                % valueType.name() % key
```

Comment:
I assume you've dropped the `BOOST_CURRENT_FUNCTION` bit here because `LOGLS_WARN` automatically includes the function name?  If not, it seems like a regression.